Review-08-Numerical-Integration-and-Differentiation.ipynb

```diff
@@ -110,8 +110,8 @@
    "source": [
     "> 8.9. Name two different methods for computing the weights corresponding to a given set of nodes of a quadrature rule.\n",
     "\n",
-    "1. Newton-Cotes rules (midpoint, trapezoid, Simpson): nodes are equally spaced and weights are fixed.\n",
-    "2. Gaussian Quadrature: nodes and weights are chosen to maximize the degree (degree is always $2n-1$)."
+    "1. Method of Undetermined Coefficients .\n",
+    "2. Polynomial Interpolation."
```

Comment:
Examples 8.1 and 8.4 give derivations of Simpson's rule and Gaussian quadrature using the method of undetermined coefficients.  I don't see any examples or references derived from polynomial interpolation apart from the description given in section 8.3.  The question could have been better written as:

> Name two different methods for deriving the weights corresponding to a given set of nodes of a quadrature rule.